--- origsrc/ftgl-2.1.3~rc5/docs/doxygen.cfg.in	2008-05-22 09:38:56.000000000 -0500
+++ src/ftgl-2.1.3~rc5/docs/doxygen.cfg.in	2013-09-22 22:55:24.790995700 -0500
@@ -594,7 +594,7 @@ EXAMPLE_RECURSIVE      = NO
 # directories that contain image that are included in the documentation (see 
 # the \image command).
 
-IMAGE_PATH             = images
+IMAGE_PATH             = @srcdir@/images
 
 # The INPUT_FILTER tag can be used to specify a program that doxygen should 
 # invoke to filter for each input file. Doxygen will invoke the filter program 
